gnomad_qc/v4/annotations/generate_freq_genomes.py

```diff
@@ -1008,11 +1007,105 @@ def generate_v4_genomes_callstats(ht: hl.Table, an_ht: hl.Table) -> hl.Table:
         freq_meta=freq_meta,
         faf_meta=faf_meta,
         faf_index_dict=make_freq_index_dict_from_meta(faf_meta),
```

Comment:
Add this
```
faf_index_dict = make_freq_index_dict_from_meta(faf_meta)
```
above the comment "# Change the 'pop' keys in the freq_meta and faf_meta arrays to 'gen_anc'."

And change this line to `faf_index_dict=faf_index_dict,`

As is we get the warning
```
WARNING (gnomad.utils.release 130): Found unexpected frequency metadata groupings: {'gen_anc'}.
```

Reply:
nvm, putting it there doesn't work, but maybe add sort_order to the call to `make_freq_index_dict_from_meta`

Reply:
or actually it just needs a `hl.literal(` around `faf_meta`